src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -19,25 +19,21 @@
   public static final String WIKI_NAME_FANDOM_SUFFIX = "fandom";
   private static final Set<String>
       FANDOM_EXCLUDED_WIKIS =
-      new HashSet<>(Arrays.asList(new String[]{"community", "gameofthrones", "sydneybuses"}));
+      new HashSet<>(Arrays.asList(new String[]{"community", "gameofthrones", "sydneybuses", "elderscrolls"}));
   private final String wikiName;
-  private final boolean areLanguageTestsEnabled;
   private Boolean forceHttps;
-  private Boolean forceLanguageInPath;
   private Boolean forceWikiaOrg;
   private String language;
 
-  private UrlBuilder(String wiki, String env, Boolean forceHttps, Boolean forceLanguageInPath, String language) {
+  private UrlBuilder(String wiki, String env, Boolean forceHttps, String language) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "UrlBuilder" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)